lib/foreman_tasks/tasks/export_tasks.rake

```diff
@@ -259,7 +259,7 @@ namespace :foreman_tasks do
 
         File.open(File.join(tmp_dir, 'index.html'), 'w') { |file| file.write(PageHelper.pagify(PageHelper.generate_index(tasks))) }
 
-        sh("tar cvzf #{export_filename} #{tmp_dir} > /dev/null")
+        sh("tar czf #{export_filename} #{tmp_dir}")
```

Comment:
Since we don't need to spawn a shell anymore, could you change this to use [Kernel#system](https://ruby-doc.org/core-2.7.0/Kernel.html#method-i-system) using the `cmdname, arg1, ...` form?

Reply:
Sure! Done.